packages/react/src/xcss/pseudos.ts

```diff
@@ -0,0 +1,27 @@
+import type { Pseudos as CSSPseudos } from 'csstype';
+// List of pseudo-classes and pseudo-elements are from csstype
+// but with & added in the front, so that we target the current element
+// (instead of a child element)
+type SafePseudos = Exclude<
```

Comment:
nit: Can we invert this and declare them ourselves so when new pseudos are introduced we can control the roll out ensuring Compiled handles it?

@JakeLane wdyt?

Reply:
Agree, we want to own these. New CSS rules seem to get used almost immediately which is a huge problem for browser support - types are a good layer to stop that

Reply:
Can have a look. Happy to invert the logic and hardcode the pseudo list as it was previously. Interestingly @liamqma had added the `&` prefix in the source of the current implementattion for `cssMap` which is partly why I pulled it out so had access to both types.

Reply:
Can change the type to "without &" for CSS Map too as long as it's consistent. @dddlr , do you remember why we choose "with &"?

Reply:
> Interestingly @liamqma had added the & prefix in the source of the current implementattion for cssMap which is partly why I pulled it out so had access to both types.

Oh @DarkPurple141 I added the type checking related stuff in cssMap haha, my cssMap changes were in the same commit as Liam's so everything got a bit murky in the commit history

&-ful and &-less forms work the same in Compiled - both the below forms target the same element (not a child element):
```
css({ ':hover': { color: 'blue' } })
css({ '&:hover': { color: 'blue' } })
```

The `(instead of a child element)` in the comment is incorrect and was written before I realised this - the comment should probably be changed tbh

I went with only allowing the form with `&` for the sake of consistency (one way to write `:hover` is better than two) and because it more clearly showed that it was targetting the same element. But happy to change it to be without `&` if that's easier for migration from emotion

Reply:
Also agree with @JakeLane about hardcoding what pseudos we allow - gives us the ability to know exactly what selectors developers will be using (without having to monitor any changes being made to `csstype`)